static/styles/settings.css

```diff
@@ -1092,17 +1092,34 @@ input[type="checkbox"] {
         border-radius: 5px;
 
         h4 {
+            .name {
+                font-weight: 1000;
```

Comment:
Previously, this was set to `bold` i.e. `700`. See https://github.com/zulip/zulip/blob/ca17a452fd270e7f6382aea09382972e818f9bca/static/styles/settings.css#L1155.